src/components/SocialButton/index.tsx

```diff
@@ -4,12 +4,15 @@ import { Button } from '@dedo_ai/gui-com-lib';
 import { type UseGoogleLoginOptionsImplicitFlow, useGoogleLogin } from '@react-oauth/google';
 import { isEmpty } from 'lodash';
 
+import useAppleAuth from './useAppleAuth';
+
 const HOOKS_MAPPER = {
   'google-hook': useGoogleLogin,
+  'apple-hook': useAppleAuth,
```

Comment:
'apple-hook': useScript;
Then pass the appleAuthHelpers.APPLE_SCRIPT_SRC as parameter from initHookCb